Command/InstallCommand.php

```diff
@@ -0,0 +1,246 @@
+    {
+        if ($this->input->getOption(self::LOAD_FIXTURES_OPTION_NAME)) {
+            $this->loadFixtures();
+        }
+
+        if ($this->input->getOption(self::CREATE_USER_OPTION_NAME)) {
+            $this->saveUserToDatabase(
+                $this->getValidator()->validateUserName($this->input->getOption(self::USER_LOGIN_OPTION_NAME)),
+                $this->getValidator()->validateUserEmail($this->input->getOption(self::USER_EMAIL_OPTION_NAME)),
+                $this->getValidator()->validateUserPassword($this->input->getOption(self::USER_PASSWORD_OPTION_NAME))
+            );
+        }
+    }
+
+    /**
+     * @param $userName
+     * @param $userEmail
+     * @param $userPassword
+     */
+    private function saveUserToDatabase($userName, $userEmail, $userPassword)
+    {
+        $userManager = $this->getContainer()->get('fos_user.user_manager');
+
+        $user = $userManager->createUser();
+        $user->setUsername($userName);
+        $user->setEmail($userEmail);
+        $user->setPlainPassword($userPassword);
+        $user->setEnabled(true);
+
+        $userManager->updateUser($user);
+    }
+
+    private function createUser()
+    {
+        list($userNameOptionValue, $userEmailOptionValue, $userPasswordOptionValue) = $this->getUserOptions();
+
+        $userNameQuestion = new Question(
+            $this->getQuestion('Please enter the user name', $userNameOptionValue),
+            $userNameOptionValue
+        );
+        $userNameQuestion->setValidator([$this->getValidator(), 'validateUserName']);
+
+        $userEmailQuestion = new Question(
+            $this->getQuestion('Please enter the user email', $userEmailOptionValue),
+            $userEmailOptionValue
+        );
+        $userEmailQuestion->setValidator([$this->getValidator(), 'validateUserEmail']);
+
+        $userPasswordQuestion = new Question(
+            $this->getQuestion('Please enter the user password', $userPasswordOptionValue),
+            $userPasswordOptionValue
+        );
+        $userPasswordQuestion->setValidator([$this->getValidator(), 'validateUserPassword']);
+
+        $userName = $this->questionHelper->ask($this->input, $this->output, $userNameQuestion);
+        $userEmail = $this->questionHelper->ask($this->input, $this->output, $userEmailQuestion);
+        $userPassword = $this->questionHelper->ask($this->input, $this->output, $userPasswordQuestion);
+
+        $this->saveUserToDatabase($userName, $userEmail, $userPassword);
+    }
+
+    private function loadFixtures()
+    {
+        $command = $this->getApplication()->find('doctrine:fixtures:load');
+        $input = new ArrayInput([]);
+        $input->setInteractive($this->input->isInteractive());
+
+        $command->run($input, $this->output);
+    }
+
+    private function printIntro()
+    {
+        $this->output->writeln([
+            '',
+            '<bg=blue;fg=white>Welcome to Grossum Installation!</>',
```

Comment:
Move this to separate method and pass text only and style settings.

See `Sensio\Bundle\GeneratorBundle\Command\Helper\QuestionHelper::writeSection`